app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -281,9 +281,11 @@ public LeakTool getLeakTool() {
 
     initializeFlurry(this, BuildConfig.FLURRY_KEY);
 
-    generateAptoideUuid().andThen(
+    generateAptoideUuid()
+        .andThen(
             Completable.mergeDelayError(initializeRakamSdk(), initializeSentry(),
                 initializeIndicative()))
+        .andThen(aptoideInstalledAppsRepository.syncWithDevice("com.appcoins.wallet"))
```

Comment:
don't we have a variable somewhere that represents the wallet package? if not I would at least extract this to a global variable in App, this package already serves many purposes, something that would provide the wallet package name would help us manage dev <-> prod envs.